environment

```diff
@@ -1,9 +1,34 @@
-#!/usr/bin/env bash
+# Resolve the location of this file and set APP_HOME to the root
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
+export APP_HOME="$(cd -P "$(dirname "$SOURCE")" && pwd)"
+
 set -a
-PROJECT_ROOT=`pwd`
-SERVICE_NAME='query'
-DCP_DNS_DOMAIN='data.humancellatlas.org'
-DEPLOYMENT_STAGE="${DEPLOYMENT_STAGE:-dev}"
-AWS_DEFAULT_REGION='us-east-1'
-API_HOST=${SERVICE_NAME}.${DEPLOYMENT_STAGE}.${DCP_DNS_DOMAIN}
+APP_NAME=dcpquery
+STAGE="${DEPLOYMENT_STAGE:-dev}"
+API_DOMAIN_NAME=query.${STAGE}.data.humancellatlas.org
```

Comment:
prod env?

Reply:
Added a special case for the production environment, thanks.